tool/menu.lua

```diff
@@ -148,36 +148,55 @@ end
 
 local function randomizermenu()
   local selected = ui.select({
+    "New Seed on New Game    " .. randomizer.newgamenewseedui[randomizer.newgamenewseed],
     "Proficiency: " .. randomizer.proficiency,
-    "Unseeded",
-    "Seeded",
+    "Random seed",
+    "Set seed",
     "Reset",
     "Back",
   })
-  if selected == 1 then
+  if selected == 1 then -- New Seed on New Game
+    local index = table.indexof(randomizer.newgamenewseedvalues, randomizer.newgamenewseed)
+    index = ((index - 1 + 1) % #randomizer.newgamenewseedvalues) + 1
+    randomizer.newgamenewseed = randomizer.newgamenewseedvalues[index]
+
+  elseif selected == 2 then -- Proficiency
     local index = table.indexof(randomizer.proficiencies, randomizer.proficiency)
     index = ((index - 1 + 1) % #randomizer.proficiencies) + 1
     randomizer.proficiency = randomizer.proficiencies[index]
-  elseif selected == 2 then
+
+  elseif selected == 3 then -- Unseeded
+    randomizer.seedqueue = {}
     randomizer.seed = ""
-    randomizer.randomize()
-    state = STATES.RANDOMIZER
-  elseif selected == 3 then
+    randomizer.seedtype = "Random seed"
```

Comment:
It might make sense to create a `SEEDTYPE` enum-ish similar to the `STATES` enum-ish here.
```lua
randomizer.SEEDTYPE = {
    RANDOMSEED = "Random Seed",
    …
}

randomizer.seedtype = randomizer.SEEDTYPE.RANDOMSEED
```